<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAjoutPortfolioItems</class>
 <widget class="QDialog" name="DialogAjoutPortfolioItems">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonAjout">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelNom">
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditNom"/>
    </item>
    <item>
     <widget class="QLabel" name="labelDesc">
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditDesc"/>
    </item>
    <item>
     <widget class="QLabel" name="labelUrlMemento">
      <property name="text">
       <string>URL Mememto :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditUrlMemento"/>
    </item>
    <item>
     <widget class="QLabel" name="labelAncre">
      <property name="text">
       <string>Ancre : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditAncre"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelImg">
        <property name="text">
         <string>Selectionner une image :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonSelectImg">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
